repos/fdbt-site/src/pages/api/ticketConfirmation.ts

```diff
@@ -1,9 +1,15 @@
 import { NextApiResponse } from 'next';
 import { NextApiRequestWithSession } from '../../interfaces/index';
-import { redirectTo, redirectToError } from '../../utils/apiUtils';
+import { getFareTypeFromFromAttributes, redirectTo, redirectToError } from '../../utils/apiUtils';
 
-export default (_req: NextApiRequestWithSession, res: NextApiResponse): void => {
+export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
     try {
+        const fareTypeAttribute = getFareTypeFromFromAttributes(req);
+
+        if (['single', 'return', 'flatFare'].includes(fareTypeAttribute)) {
+            redirectTo(res, '/selectCaps');
+            return;
```

Comment:
Dont we need to be doing a check here to see if we should be redirecting straight to /selectPurchaseMethods?